change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "marked barCode capability beta for potential future changes",
```

Comment:
Please change this to `Added beta tag to `barCode` capability to accurately reflect level of support offered`

Reply:
Ugh, backticks around `barCode`, not the rest :)